src/api/data/util/util.go

```diff
@@ -0,0 +1,12 @@
+// Package util contains small utilities used in other packages
+package util
+
+// Int64ToPtr converts an int64 to an *int64
+func Int64ToPtr(n int64) *int64 {
+	return &n
+}
+
+// StrToPtr converts a string to a *string
+func StrToPtr(s string) *string {
```

Comment:
creating packages is cheap.

`util` is a particularly bad name for a package, for a few reasons:

1. by definition, it means that the bunch of thing you put in there has not much in common besides being generally useful. The family of `FooPtr` functions here have a lot in common: they take a Foo and return a *Foo, for the purpose of taking the address of a literal which is not allowed by language spec.

2. without `goimports` programming in Go would be such a pain. `goimports` work by assuming that there is signal in the combination of packagename+function signature. It doesn't work well when a lot of different packages have both the same name and same functions (like the often happens with the various log libraries) and obviously it doesn't work well you need code from multiple packages that are named in the same way (something that usually doesn't happen with the aforementioned log libraries). This makes `util` a choice of last resort, because abusing it will eventually lead to having many `util` packages spread in your code base and in the libraries you depend on and thus a kitten will die every time you'll have to manually import or manually set an alias of an import and/or rename a bunch of package aliases in a code snippet that you're moving around the codebase just because of a conflict with the `util` name. Ah, and some people call it `utils` ... `utils.Foo + util.Bar`